cli/cmd/checkRunner.go

```diff
@@ -205,13 +206,13 @@ func checkInstallResourcesExist(ctx context.Context, kubernetesProvider *kuberne
 	installResourcesExist := checkResourceExist(kubernetes.DaemonRoleName, "role", exist, err)
 
 	exist, err = kubernetesProvider.DoesRoleBindingExist(ctx, config.Config.MizuResourcesNamespace, kubernetes.DaemonRoleBindingName)
-	installResourcesExist = checkResourceExist(kubernetes.DaemonRoleBindingName, "role binding", exist, err)
+	installResourcesExist = installResourcesExist && checkResourceExist(kubernetes.DaemonRoleBindingName, "role binding", exist, err)
```

Comment:
I'm little confused from this line,
the idea is to not check the current resource and if the previous is not exists?

Reply:
so from the moment that something not exists you will not query for more resources?